kernel_module/uio/uio_ivshmem.c

```diff
@@ -15,6 +16,8 @@
 #include <linux/uio_driver.h>
 #include <linux/io.h>
 
+#include <linux/version.h>
```

Comment:
please add this right under the other includes, some people like to order them somehow but i do not care